### ISSUE TITLE
1.11.1 regression for JrJackson adapter for v0.2.7 and lower

### ISSUE THREAD

**Issue by jwinter** [intridea/multi_json#158]
Commit 3c730fd provides a second arg to JrJackson.dump:

but JrJackson.dump only takes 1 arg in 0.2.7 and lower:
https://github.com/guyboertje/jrjackson/blob/v0.2.7/lib/jrjackson/jrjackson.rb#L34

```
[9] pry(main)> MultiJson.dump({ ok: false })
ArgumentError: wrong number of arguments calling `dump` (2 for 1)
from /Users/Joseph.Winter/.rvm/gems/jruby-1.7.18@psg/gems/multi_json-1.11.1/lib/multi_json/adapters/jr_jackson.rb:15:in `dump'
```

I think this can be fixed by requiring JrJackson >= 0.2.8, but at least in 0.2.7 JrJackson.dump only accepts one argument.

**Comment by rwz** [comment]
Can you test it with current master please? I think I've fixed it, and the tests are green, but it's always better be safe than sorry. :)

**Comment by jwinter** [comment]
Current master now works with 0.2.7 JrJackson in my local testing. Thanks for the fix, @rwz !

**Comment by rwz** [comment]
Released 1.11.2
